data/notices.json

```diff
@@ -423,6 +423,18 @@
         }
       ],
       "schemaVersion": "1"
+    },
+    {
+      "title": "(aws-events): EventBus fails because of change to logical id without modifiying the statement id",
+      "issueNumber": 28520,
+      "overview": "#27340 introduced the ability to create multiple event bus policies on a single event bus. To facilitate this, the logical Id was changed from \"Policy\" to the statementId. This triggers a replacement, which fails in CloudFormation because the statement ID does not change.",
```

Comment:
#27340 isn't going to mean anything to the people who view this (when they cdk synth their apps). can you rewrite this overview to be self-sufficient